packages/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -51,7 +51,7 @@ export class MongoDBCorePlugin extends BasePlugin<typeof mongodb> {
 
     if (this._moduleExports.Server) {
       for (const fn of this._SERVER_METHODS) {
-        this._logger.debug(`patching mongodb-core.Server.prototype.${fn}`);
+        this._logger.debug(`patching mongodb.Server.prototype.${fn}`);
```

Comment:
A little misleading since we are actually patching `mongodb-nodejs` and not `mongodb`

Reply:
We are patching `mongodb`, i can't find on the spec that the `COMPONENT` property should be the same as the module so i believed that it was fine like this

Reply:
I think the `_COMPONENT` property is not part of the spec and just something we've been doing by convention. Same goes with the `moduleName`. the `component` span property is the library being instrumented.

Reply:
Thats fine for me, i will update that